src/Adapter/Presenter/Object/ObjectPresenter.php

```diff
@@ -54,19 +53,8 @@ public function present($object)
             $presentedObject[$fieldName] = $object->{$fieldName};
         }
 
-        if ($object instanceof Product) {
```

Comment:
This key was not used in the core anywhere, but I added it back to productcontroller just to be sure.